goodtablesio/blueprints/github.py

```diff
@@ -70,25 +66,22 @@ def clone_repo(task_id, clone_url):
     return clone_dir
 
 
-def get_files_to_validate(clone_dir):
-    paths = get_dir_paths(clone_dir)
-    return get_tabular_file_paths(paths)
+def _get_task_conf(clone_url):
+    pattern = r'github.com/(?P<user>[^/]*)/(?P<repo>[^/]*)\.git'
+    match = re.search(pattern, clone_url)
+    user = match.group('user')
+    repo = match.group('repo')
+    template = 'https://raw.githubusercontent.com/{user}/{repo}/master/goodtables.yml'
```

Comment:
Also for private repos we will need an `Authorization` header, but that's definitely post-MVP